### ISSUE TITLE
 Got status 17 from selenium

### ISSUE THREAD

**Issue by jagdheesh-github** [angular/webdriver-manager#417]
I am using protractor : 5.6.2
web driver manager : 12.1.6
selenium-server-standalone : 3.141.59.
chrome browser version : 76.0.3809.100
chrome driver version launched by webdriver manager is :76.0.3809.12 
my protractor - jasmine tests are failing upon reaching a angular page. this did not happen before. 

it returns :  

Angular testability undefined  
or the below error from selenium

..[16:28:20] E/BlockingProxy - Got status 17 from selenium { additionalInformation: '\nDriver info: driver.version: unknown',
  localizedMessage:
   'javascript error: circular reference\n  (Session info: chrome=76.0.3809.100)\nBuild info: version: \'3.141.59\', revision: \'e82be7d358\', time: \'2018-11-14T08:25:53\'\nSystem info: host: \'ALIPL0997\', ip: \'192.168.225.58\', os.name: \'Windows 10\', os.arch: \'amd64\', os.version: \'10.0\', java.version: \'1.8.0_221\'\nDriver info: driver.version: unknown',
  systemInformation:
   'System info: host: \'ALIPL0997\', ip: \'192.168.225.58\', os.name: \'Windows 10\', os.arch: \'amd64\', os.version: \'10.0\', java.version: \'1.8.0_221\'',
  supportUrl: null,
  cause: null,
  stackTrace:
   [ null,
     null,
     null,
     null,
     null,
     null,
     null,
     null,
     null,
     null,
     null,
     null,
     null,
     null,
     null ],
  suppressed: [],
  message:
   'javascript error: circular reference\n  (Session info: chrome=76.0.3809.100)\nBuild info: version: \'3.141.59\', revision: \'e82be7d358\', time: \'2018-11-14T08:25:53\'\nSystem info: host: \'ALIPL0997\', ip: \'192.168.225.58\', os.name: \'Windows 10\', os.arch: \'amd64\', os.version: \'10.0\', java.version: \'1.8.0_221\'\nDriver info: driver.version: unknown',

**Comment by TravkinAlex** [comment]
Correct, chromedriver version *.12 has this bug which is fixed in later versions;
however, `webdriver-manager` currently **ignores the patch version** of chromedriver, and you have to use some workarounds to install the latest one - see issue https://github.com/angular/webdriver-manager/issues/408, I've actually submitted a PR to fix this and referenced it there.
